### ISSUE TITLE
NoClassDefFoundError version 0.7.0 on Forge 10.13.4.1614 for Minecraft 1.7.10

### ISSUE THREAD

**Issue by realStutz** [Wehavecookies56/Kingdom-Keys-1.7-#27]
I try to launch Minecraft with the mod installed and get the following message:

**The game crashed whilst there was a severe problem during mod loading that has caused the game to fail
Error: cpw.mods.fml.common.LoaderException: java.lang.NoClassDefFoundError: wehavecookies56/kk/item/keyblades/ItemVulpeus**

Here is the full crash report:

---- Minecraft Crash Report ----
// Who set us up the TNT?

Time: 4/26/18 4:10 PM
Description: There was a severe problem during mod loading that has caused the game to fail

cpw.mods.fml.common.LoaderException: java.lang.NoClassDefFoundError: wehavecookies56/kk/item/keyblades/ItemVulpeus
	at cpw.mods.fml.common.LoadController.transition(LoadController.java:163)
	at cpw.mods.fml.common.Loader.preinitializeMods(Loader.java:559)
	at cpw.mods.fml.client.FMLClientHandler.beginMinecraftLoading(FMLClientHandler.java:243)
	at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java:480)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:878)
	at net.minecraft.client.main.Main.main(SourceFile:148)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
Caused by: java.lang.NoClassDefFoundError: wehavecookies56/kk/item/keyblades/ItemVulpeus
	at wehavecookies56.kk.item.AddedItems.initKeyBlades(AddedItems.java:106)
	at wehavecookies56.kk.KingdomKeys.preInit(KingdomKeys.java:150)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at cpw.mods.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:532)
	at sun.reflect.GeneratedMethodAccessor2.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
	at com.google.common.eventbus.EventBus.post(EventBus.java:275)
	at cpw.mods.fml.common.LoadController.sendEventToModContainer(LoadController.java:212)
	at cpw.mods.fml.common.LoadController.propogateStateMessage(LoadController.java:190)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
	at com.google.common.eventbus.EventBus.post(EventBus.java:275)
	at cpw.mods.fml.common.LoadController.distributeStateMessage(LoadController.java:119)
	at cpw.mods.fml.common.Loader.preinitializeMods(Loader.java:556)
	... 10 more
Caused by: java.lang.ClassNotFoundException: wehavecookies56.kk.item.keyblades.ItemVulpeus
	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:191)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 38 more
Caused by: java.lang.NoClassDefFoundError: mods/battlegear2/api/weapons/IBattlegearWeapon
	at java.lang.ClassLoader.defineClass1(Native Method)
	at java.lang.ClassLoader.defineClass(ClassLoader.java:760)
	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142)
	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:182)
	... 40 more
Caused by: java.lang.ClassNotFoundException: mods.battlegear2.api.weapons.IBattlegearWeapon
	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:101)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 44 more

A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- System Details --
Details:
	Minecraft Version: 1.7.10
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_25, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 351417472 bytes (335 MB) / 805306368 bytes (768 MB) up to 1073741824 bytes (1024 MB)
	JVM Flags: 8 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx1G -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=16M
	AABB Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
	FML: MCP v9.05 FML v7.10.99.99 Minecraft Forge 10.13.4.1614 103 mods loaded, 103 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
	UCH	mcp{9.05} [Minecraft Coder Pack] (minecraft.jar) 
	UCH	FML{7.10.99.99} [Forge Mod Loader] (forge-1.7.10-10.13.4.1614-1.7.10.jar) 
	UCH	Forge{10.13.4.1614} [Minecraft Forge] (forge-1.7.10-10.13.4.1614-1.7.10.jar) 
	UCH	AM2-Preloader{0.0.3} [AMCore] (minecraft.jar) 
	UCH	appliedenergistics2-core{rv3-beta-6} [Applied Energistics 2 Core] (minecraft.jar) 
	UCH	CodeChickenCore{1.0.7.47} [CodeChicken Core] (minecraft.jar) 
	UCH	Micdoodlecore{} [Micdoodle8 Core] (minecraft.jar) 
	UCH	NotEnoughItems{1.0.5.120} [Not Enough Items] (NotEnoughItems-1.7.10-1.0.5.120-universal.jar) 
	UCH	OpenComputers|Core{1.7.2.1166} [OpenComputers (Core)] (minecraft.jar) 
	UCH	ThE-core{1.0.0.1} [Thaumic Energistics Core] (minecraft.jar) 
	UCH	ThaumicTinkerer-preloader{0.1} [Thaumic Tinkerer Core] (minecraft.jar) 
	UCH	voltzenginepreloader{0.0.1} [Voltz Engine Preloader] (minecraft.jar) 
	UCH	WitchingGadgetsCore{1.1.10} [Witching Gadgets Core] (minecraft.jar) 
	UCH	<CoFH ASM>{000} [CoFH ASM] (minecraft.jar) 
	UCH	mod_ThreadedLighting{1.7.10-1.0} [Threaded Lighting] (minecraft.jar) 
	UCH	MC3ASMCore{0.1} [Metroid Cubed 3 ASM Core] (minecraft.jar) 
	UCH	extendedshaders{6.0} [Extended Shaders] ([1.7.10]+Extended+Shaders+6.0.jar) 
	UCH	gilded-games-util{1.7.10-1.2} [Gilded Games Utility] (gilded-games-util-1.7.10-1.9.jar) 
	UCH	AnimationAPI{1.2.4} [AnimationAPI] (AnimationAPI-1.7.10-1.2.4.jar) 
	UCH	arsmagica2{1.4.0.009} [Ars Magica 2] (1.7.10_AM2-1.4.0.009.jar) 
	UCH	CoFHCore{1.7.10R3.1.4} [CoFH Core] (CoFHCore-[1.7.10]3.1.4-329.jar) 
	UCH	BuildCraft|Core{7.1.23} [BuildCraft] (buildcraft-7.1.23.jar) 
	UCH	BuildCraft|Energy{7.1.23} [BC Energy] (buildcraft-7.1.23.jar) 
	UCH	IC2{2.2.823-experimental} [IndustrialCraft 2] (industrialcraft-2-2.2.823-experimental.jar) 
	UCH	Mantle{1.7.10-0.3.2.jenkins191} [Mantle] (Mantle-1.7.10-0.3.2.jenkins191.jar) 
	UCH	Waila{1.5.10} [Waila] (Waila-1.5.10_1.7.10.jar) 
	UCH	TConstruct{1.7.10-1.8.8.build988} [Tinkers' Construct] (TConstruct-1.7.10-1.8.8.jar) 
	UCH	GalacticraftCore{3.0.12} [Galacticraft Core] (GalacticraftCore-1.7-3.0.12.504.jar) 
	UCH	mc3{0.3.33.1} [Metroid Cubed 3] ([1.7.10]+Metroid+Cubed+3+0.3.33.1-server_hotfix.jar) 
	UCH	Baubles{1.0.1.10} [Baubles] (Baubles-1.7.10-1.0.1.10.jar) 
	UCH	Thaumcraft{4.2.3.5} [Thaumcraft] (Thaumcraft-1.7.10-4.2.3.5.jar) 
	UCH	ForgeMultipart{1.2.0.347} [Forge Multipart] (ForgeMultipart-1.7.10-1.2.0.347-universal.jar) 
	UCH	appliedenergistics2{rv3-beta-6} [Applied Energistics 2] (appliedenergistics2-rv3-beta-6.jar) 
	UCH	ThaumicTinkerer{unspecified} [Thaumic Tinkerer] (ThaumicTinkerer-2.5-1.7.10-164.jar) 
	UCH	ForbiddenMagic{1.7.10-0.575} [Forbidden Magic] (Forbidden+Magic-1.7.10-0.575.jar) 
	UCH	EnderStorage{1.4.7.37} [EnderStorage] (EnderStorage-1.7.10-1.4.7.37-universal.jar) 
	UCH	OpenComputers{1.7.2.1166} [OpenComputers] (OpenComputers-MC1.7.10-1.7.2.1166-universal.jar) 
	UCH	DummyCore{2.0.1710.0.A} [DummyCore] (DummyCore-2.0.1710.0A.jar) 
	UCH	thaumicbases{1.3.1710.2} [Thaumic Bases] (ThaumicBases-1.3.1710.4.jar) 
	UCH	ThaumicHorizons{1.1.9} [Thaumic Horizons] (thaumichorizons-1.7.10-1.1.9.jar) 
	UCH	AdvancedThaumaturgy{2.0} [Advanced Thaumaturgy] (AdvancedThaumaturgy-2.0-1.7.10-30.jar) 
	UCH	aether{1.7.10-1.6} [Aether II] (aether-1.7.10-1.6.jar) 
	UCH	AetherAspects{Alpha 1.7.10-1.2} [Aether Aspects] (AetherAspects-1.7.10-1.2.jar) 
	UCH	Automagy{0.28.2} [Automagy] (Automagy-1.7.10-0.28.2.jar) 
	UCH	Avaritia{1.13} [Avaritia] (Avaritia-1.13.jar) 
	UCH	Forestry{4.2.16.64} [Forestry for Minecraft] (forestry_1.7.10-4.2.16.64.jar) 
	UCH	BinnieCore{2.0.22.7} [Binnie Core] (binnie-mods-1.7.10-2.0.22.7.jar) 
	UCH	Botany{2.0.22.7} [Botany] (binnie-mods-1.7.10-2.0.22.7.jar) 
	UCH	ExtraTrees{2.0.22.7} [Extra Trees] (binnie-mods-1.7.10-2.0.22.7.jar) 
	UCH	Genetics{2.0.22.7} [Genetics] (binnie-mods-1.7.10-2.0.22.7.jar) 
	UCH	ExtraBees{2.0.22.7} [Extra Bees] (binnie-mods-1.7.10-2.0.22.7.jar) 
	UCH	BuildCraft|Transport{7.1.23} [BC Transport] (buildcraft-7.1.23.jar) 
	UCH	BuildCraft|Factory{7.1.23} [BC Factory] (buildcraft-7.1.23.jar) 
	UCH	BuildCraft|Silicon{7.1.23} [BC Silicon] (buildcraft-7.1.23.jar) 
	UCH	BuildCraft|Robotics{7.1.23} [BC Robotics] (buildcraft-7.1.23.jar) 
	UCH	BuildCraft|Builders{7.1.23} [BC Builders] (buildcraft-7.1.23.jar) 
	UCH	ChickenChunks{1.3.4.19} [ChickenChunks] (ChickenChunks-1.7.10-1.3.4.19-universal.jar) 
	UCH	CompactSolars{4.4.39.315} [Compact Solar Arrays] (CompactSolars-1.7.10-4.4.39.315-universal.jar) 
	UCH	craftingpillars{1.6.0} [Crafting Pillars Mod] (CraftingPillars-1.7.x-1.6.1.jar) 
	UCH	extracells{2.3.14} [Extra Cells 2] (ExtraCells-1.7.10-2.3.14b200.jar) 
	UCH	GalacticraftMars{3.0.12} [Galacticraft Planets] (Galacticraft-Planets-1.7-3.0.12.504.jar) 
	UCH	ExtraPlanets{1.7.10-2.0.1} [Extra Planets] (ExtraPlanets-1.7.10-2.0.1.jar) 
	UCH	ForgeMicroblock{1.2.0.347} [Forge Microblocks] (ForgeMultipart-1.7.10-1.2.0.347-universal.jar) 
	UCH	McMultipart{1.2.0.347} [Minecraft Multipart Plugin] (ForgeMultipart-1.7.10-1.2.0.347-universal.jar) 
	UCH	gadomancy{1.0.7.3} [Gadomancy] (gadomancy-1.7.10-1.0.7.3.jar) 
	UCH	GalaxySpace{1.2.1} [GalaxySpace] (GalaxySpace-1.2.1+STABLE.jar) 
	UCH	GollumCoreLib{2.0.0} [Gollum Core Lib] (GollumCoreLib-2.0.0-1.7.10.jar) 
	UCH	iChunUtil{4.2.3} [iChunUtil] (iChunUtil-4.2.3.jar) 
	UCH	inventorytweaks{1.59-dev-152-cf6e263} [Inventory Tweaks] (InventoryTweaks-1.59-dev-152.jar) 
	UCE	kk{0.7.0.1.7.10} [Kingdom Keys] (Kingdom+Keys-0.7.0.jar) 
	UCH	magicalcrops{4.0.0_PUBLIC_BETA_3} [Magical Crops] (magicalcrops-4.0.0_PUBLIC_BETA_3.jar) 
	UCH	mixedmagic{1.0.1710.2} [Mixed Magic] (MixedMagic-1.0.1710.2.jar) 
	UCH	MorePistons{2.0.0} [More Pistons] (MorePistons-2.0.0-1.7.10.jar) 
	UCH	Morph{0.9.3} [Morph] (Morph-Beta-0.9.3.jar) 
	UCH	NEIAddons{1.12.14.40} [NEI Addons] (neiaddons-1.12.14.40-mc1.7.10.jar) 
	UCH	NEIAddons|Developer{1.12.14.40} [NEI Addons: Developer Tools] (neiaddons-1.12.14.40-mc1.7.10.jar) 
	UCH	NEIAddons|AppEng{1.12.14.40} [NEI Addons: Applied Energistics 2] (neiaddons-1.12.14.40-mc1.7.10.jar) 
	UCH	NEIAddons|Botany{1.12.14.40} [NEI Addons: Botany] (neiaddons-1.12.14.40-mc1.7.10.jar) 
	UCH	NEIAddons|Forestry{1.12.14.40} [NEI Addons: Forestry] (neiaddons-1.12.14.40-mc1.7.10.jar) 
	UCH	NEIAddons|CraftingTables{1.12.14.40} [NEI Addons: Crafting Tables] (neiaddons-1.12.14.40-mc1.7.10.jar) 
	UCH	NEIAddons|ExNihilo{1.12.14.40} [NEI Addons: Ex Nihilo] (neiaddons-1.12.14.40-mc1.7.10.jar) 
	UCH	neiintegration{1.1.2} [NEI Integration] (NEIIntegration-MC1.7.10-1.1.2.jar) 
	UCH	stefinusguns{0.5.2} [New Stefinus Guns] (New-Stefinus-3D-Guns-Mod-1.7.10.jar) 
	UCH	recipehandler{1.7.10} [NoMoreRecipeConflict] (NoMoreRecipeConflict-0.3(1.7.10).jar) 
	UCH	fodc{1.6.1} [Ore Dictionary Converter] (OreDictionaryConverter-1.6.1.jar) 
	UCH	ProjectE{1.7.10-PE1.10.1} [ProjectE] (ProjectE-1.7.10-PE1.10.1.jar) 
	UCH	TaintedMagic{r7.82} [Tainted Magic] (Tainted-Magic-r7.82.jar) 
	UCH	thaumcraftgates{1.4.2} [Thaumcraft Gates] (ThaumcraftGates-1.4.2.jar) 
	UCH	thaumcraftneiplugin{@VERSION@} [Thaumcraft NEI Plugin] (thaumcraftneiplugin-1.7.10-1.7a.jar) 
	UCH	thaumicenergistics{1.1.2.0} [Thaumic Energistics] (thaumicenergistics-1.1.2.0.jar) 
	UCH	elysium{1.0} [The Elysium] (TheElysium-1.7.x-1.0.3.jar) 
	UCH	TravellersGear{1.16.6} [Traveller's Gear] (TravellersGear-1.7.10-1.16.6.jar) 
	UCH	voltzengine{1.9.14.302} [Voltz Engine] (VoltzEngine-1.7.10-1.9.14b302.jar) 
	UCH	voltzenginemodcompat{1.9.14.302} [Voltz Engine Mod Compatibility Loader] (VoltzEngine-1.7.10-1.9.14b302.jar) 
	UCH	voltzenginemodflag{1.9.14.302} [VoltzEngine mod protection, flag, and region system] (VoltzEngine-1.7.10-1.9.14b302.jar) 
	UCH	WarpTheory{1.7.10R1.0} [WarpTheory] (WarpTheory-1.7-1.0-24.jar) 
	UCH	witchery{0.24.1} [Witchery] (witchery-1.7.10-0.24.1.jar) 
	UCH	WitchingGadgets{1.1.10} [Witching Gadgets] (WitchingGadgets-1.7.10-1.1.10.jar) 
	UCH	WR-CBE|Core{1.4.1.9} [WR-CBE Core] (WR-CBE-1.7.10-1.4.1.9-universal.jar) 
	UCH	WR-CBE|Addons{1.4.1.9} [WR-CBE Addons] (WR-CBE-1.7.10-1.4.1.9-universal.jar) 
	UCH	WR-CBE|Logic{1.4.1.9} [WR-CBE Logic] (WR-CBE-1.7.10-1.4.1.9-universal.jar) 
	UCH	XaeroMinimap{1.15} [Xaero's Minimap] (Xaeros_Minimap_1.15_Forge_1.7.10.jar) 
	UCH	XaeroWorldMap{1.2.4} [Xaero's World Map] (XaerosWorldMap_1.2.4_Forge_1.7.10.jar) 
	GL info: ' Vendor: 'ATI Technologies Inc.' Version: '4.5.13421 Compatibility Profile Context 16.101.1311.1001' Renderer: 'AMD Radeon(TM) R5 Graphics'
	CoFHCore: -[1.7.10]3.1.4-329
	Mantle Environment: Environment healthy.
	TConstruct Environment: Environment healthy.
	AE2 Version: beta rv3-beta-6 for Forge 10.13.4.1448
	[DummyCore]: 'Special case ASM modification mods: ''Note, that this mods might not be involved in the crash in ANY WAY!''DummyCore just prints some known mods for a lot of ASM modifications''DummyCore''CoFHCore''CoFHCore''CodeChickenCore'

Does anybody know how I can fix this???

**Comment by realStutz** [comment]
I reinstalled Battlegear 2; here's the new crash report: 
[crash-2018-04-26_16.26.31-client.txt](https://github.com/Wehavecookies56/Kingdom-Keys-1.7-8-/files/1953057/crash-2018-04-26_16.26.31-client.txt)

**Comment by realStutz** [comment]
Edit: I'm going to try uninstalling Kingdom Keys and see if that helps.

**Comment by realStutz** [comment]
FINAL EDIT: Removed Kingdom Keys AND Battlegear 2; seems to be working fine now. It appears to just be a problem with the mod itself. Seeing as the game works now, I will be closing this issue. I urge anyone else that encounters this issue follow in my footsteps by deleting and uninstalling the mod. I have included the launcher logs for further reading.
[GollumCoreLib.log](https://github.com/Wehavecookies56/Kingdom-Keys-1.7-8-/files/1953088/GollumCoreLib.log)
[latest.log](https://github.com/Wehavecookies56/Kingdom-Keys-1.7-8-/files/1953089/latest.log)
